core/app/models/workarea/order/item.rb

```diff
@@ -147,13 +147,32 @@ def customized?
       customizations.present?
     end
 
+    # Determine whether the additional fields on this item are
+    # equivalent to the passed-in Hash of parameters. Uses the
+    # `distinct_order_item_attributes` configuration setting to filter
+    # down which parameters it will be checking equivalence on.
+    #
+    # @param [Hash] params - Updated order item parameters.
+    # @return [Boolean] whether this item's attributes is equal to the
+    #                   passed-in parameters.
+    def attributes_eql?(params = {})
+      equivs = Workarea.config.distinct_order_item_attributes.map(&:to_sym)
+      attrs = params&.symbolize_keys&.slice(*equivs) || {}
+      customizations = params[:customizations]&.stringify_keys!
```

Comment:
So in reading this closer, two things:
*  `params` being `nil` is handled on line 160, but not 161, so that's inconsistent. 
* `params[:customizations]` is getting mutated with `stringify_keys!` and it's always a good idea to avoid mutating something passed in if reasonable.

A cleaner approach handling this whole situation could be:

```suggestion
    def attributes_eql?(params = {})
      params = params.to_h.with_indifferent_access
      attrs = params.slice(*Workarea.config.distinct_order_item_attributes)
      customizations = params[:customizations]&.stringify_keys
```